### DISCUSSION TITLE
1.2.0

### DISCUSSION THREAD

**Discussion by gt-john** [XMinerTech/MochaMiner#2] [Announcements]
1. 解决部分矿池被封禁的问题
2. 增加矿池列表
3. 完善AMD显卡支持
4. 集成最新内核

<hr /><em>This discussion was created from the release <a href='https://github.com/XMinerTech/XMiner/releases/tag/'>1.2.0</a>.</em>

**Comment by cloth0117** [comment]
安装后打不开啊  最新1.2.5也打不开
